receiver/splunkenterprisereceiver/scraper.go

```diff
@@ -67,7 +67,7 @@ func errorListener(ctx context.Context, eQueue <-chan error, eOut chan<- *scrape
 				eOut <- errs
 				return
 			}
-			errs.Add(err)
+			errs.AddPartial(1, err)
```

Comment:
In my understanding, the value provided to AddPartial is intended to represent metrics that failed to scrape. 

As I believe different scrape functions in the receiver could result in different numbers of missed metrics, could this pattern be enhanced to send an object with both error & count on eQueue, providing the ability to more accurately count the partial failures?

Reply:
I believe this code does accurately capture the number of failed scrapes per scrape cycle.